app/assets/javascripts/modules/track-email-alert-signup-choices.js

```diff
@@ -0,0 +1,43 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  'use strict'
+
+  var $ = global.jQuery
+
+  GOVUK.Modules.TrackEmailAlertSignupChoices = function () {
+    this.start = function (element) {
+      track(element)
+    }
+
+    function track (element) {
+      element.on('submit', function (event) {
+        var $checkedOption, eventLabel, options
+        var $submittedForm = $(event.target)
+        var $checkedOptions = $submittedForm.find('input:checked')
+        var category = $submittedForm.data('track-category')
+        var action = $submittedForm.data('track-action')
+
+        if ($checkedOptions.length) {
+          $checkedOptions.each(function (index) {
```

Comment:
Are you expecting to need a loop here? I thought the things being were tracked are radio buttons.